src/main/java/com/revature/services/BatchLogic.java

```diff
@@ -13,6 +14,6 @@
 	Page<Batch> getAllBatches(Pageable pageable);
 	void createBatch(Batch batchName);
 	void updateBatch(Batch batchName);
-	
+	Page<Person> getAllPeopleByBatchId(Pageable pageable, Integer Id);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS00117)